cli/command/clean.go

```diff
@@ -0,0 +1,117 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2021-10-15
+ * Author: Jingli Chen (Wine93)
+ */
+
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/opencurve/curveadm/cli/cli"
+	"github.com/opencurve/curveadm/internal/configure"
+	"github.com/opencurve/curveadm/internal/tasks"
+	tui "github.com/opencurve/curveadm/internal/tui/common"
+	cliutil "github.com/opencurve/curveadm/internal/utils/cli"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cleanExample = `Examples:
+  $ curveadm clean                               # Clear everything for all services
+  $ curveadm clean --only='log,data'             # Clean log and data for all services
+  $ curveadm clean --role=etcd --only=container  # Clean container for etcd services`
+)
+
+type cleanOptions struct {
+	id   string
+	role string
+	host string
+	only []string
+}
+
+func NewCleanCommand(curveadm *cli.CurveAdm) *cobra.Command {
```

Comment:
how do this deal with a misoperation? should we add some double check options? just like ceph delete pool cmd: ceph osd pool delete $pool $pool --yes-i-really-mean-it

Reply:
or add a interactive, the operator must input a 'YES' to clean a cluster or service?

Reply:
> or add a interactive, the operator must input a 'YES' to clean a cluster or service?

yep, i will improve it.